code/.eslintrc.json

```diff
@@ -126,6 +126,7 @@
     "react/jsx-indent": "off",
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
-    "semi": "off"
+    "semi": "off",
```

Comment:
What does this do? :)

Reply:
I turned off the rule `no-promise-executor-return` to make the loader not cause issues with the linter

https://eslint.org/docs/latest/rules/no-promise-executor-return